cloudformation/campaign-central.yaml

```diff
@@ -156,11 +151,11 @@ Resources:
           Action:
           - s3:GetObject
           Resource:
-          - arn:aws:s3:::github-team-keys/*
+          - arn:aws:s3:::github-public-keys/*
```

Comment:
Does this exist?

Reply:
It's the new supported version that is in the central deploy tools account rather than the EdTools account.